src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-flags-modal/osd-flags-modal.component.scss

```diff
@@ -0,0 +1,10 @@
+.osd-modal {
+  margin: 10px 0px;
```

Comment:
You can remove both this rules and just keep the `.oa-hr-small`.
The others are already taken care of by bootstrap.

Reply:
Ack